cmd/snap-update-ns/user.go

```diff
@@ -27,8 +27,14 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
+// UserProfileUpdateContext contains information about update to per-user mount namespace.
+type UserProfileUpdateContext struct {
+	CommonProfileUpdateContext
+}
+
 func applyUserFstab(snapName string) error {
-	desiredProfilePath := fmt.Sprintf("%s/snap.%s.user-fstab", dirs.SnapMountPolicyDir, snapName)
+	upCtx := &UserProfileUpdateContext{}
+	desiredProfilePath := desiredUserProfilePath(snapName)
 	desired, err := osutil.LoadMountProfile(desiredProfilePath)
```

Comment:
shouldn't this be already reorganized to be upCtx.LoadDesiredProfile() ?

Reply:
This is done so in the subsequent patches. I will bring it from there as soon as this lands.